integration/create_test.go

```diff
@@ -156,12 +158,33 @@ func TestSociCreate(t *testing.T) {
 			name:           "test create for drupal:latest",
 			containerImage: "drupal:latest",
 		},
+		// The following two tests guarantee that we have tested at least
```

Comment:
How about making 3 test cases?
1. Create for "linux/arm64"
2. Create for "linux/amd64"
3. Create for all

Reply:
See below about the all platforms tests